dev-packages/e2e-tests/test-applications/react-router-6/package.json

```diff
@@ -43,6 +44,7 @@
   "devDependencies": {
     "@playwright/test": "^1.44.1",
     "@sentry-internal/test-utils": "link:../../../test-utils",
+    "concurrently": "^8.2.2",
```

Comment:
We use `npm-run-all2` for this in the rest of the repo, so I'd say let's use the same here too, for consistency? (then you can do `run-p xx xx`)